book_source/adv_user_guide_web/SDA_Methods.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "SDA_Methods"
```

Comment:
@araiho Where do you want this file to show up? Is this supposed to supplement the information in `02_sda.Rmd` or do you want it to show up somewhere else?